src/main/java/kostka/moviecatalog/service/rabbitmq/RabbitMqReceiver.java

```diff
@@ -73,7 +75,11 @@ public void receiveMessageLatestMoviesQueue() {
         if (latestMovies.isEmpty()) {
             return;
         }
-        redisService.updateLatestMovies(latestMovies);
+        try {
+            redisService.updateMoviesInRedis(latestMovies, LATEST_MOVIES_KEY);
+        } catch (JsonProcessingException e) {
+            LOGGER.error(CANNOT_PARSE_JSON, e);
+        }
         stompService.sendSTOMPToUpdateLatestMovies();
```

Comment:
Má pořád cenu signalizovat FE, pokud se ten update neprovede?

Reply:
ajo, to da rozum! :) díky